src/components/Card/index.examples.js

```diff
@@ -32,6 +32,191 @@ storiesOf('Static: Cards')
+                  <div className='truncate'>Sync Requests with RxJS and Angular</div>
+                  <div className='w3 ml3 tr o-60'>22:22</div>
+                </div>
+              </li>
+              <li className='flex items-start relative f6 lh-solid pointer pv3 pl4 pr3 gray hover-bg-white card-progress-list-item viewed'>
+                <img src={imgJs} className='ml2 mt1' alt=''/>
+                <div className='ml2 flex flex-grow-1 lh-copy overflow-hidden lesson-title'>
+                  <div className='truncate'>Understand the Angular 2 States of</div>
+                  <div className='w3 ml3 tr o-60'>22:22</div>
+                </div>
+              </li>
+              <li className='flex items-start relative f6 lh-solid pointer pv3 pl4 pr3 gray hover-bg-white card-progress-list-item'>
+                <img src={imgRx} className='ml2 mt1' alt=''/>
+                <div className='ml2 flex flex-grow-1 lh-copy overflow-hidden lesson-title'>
+                  <div className='truncate'>Sync Requests with RxJS and Angular</div>
+                  <div className='w3 ml3 tr o-60'>22:22</div>
+                </div>
+              </li>
+              <li className='flex items-start relative f6 lh-solid pointer pv3 pl4 pr3 gray hover-bg-white card-progress-list-item'>
+                <img src={imgAngular} className='ml2 mt1' alt=''/>
+                <div className='ml2 flex flex-grow-1 lh-copy overflow-hidden lesson-title'>
+                  <div className='truncate'>Sync Requests with RxJS and Angular</div>
+                  <div className='w3 ml3 tr o-60'>22:22</div>
+                </div>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+
+    </div>
+  ))
+
+  .addWithInfo('Course Card Expanded Horizontal', () => (
+    <div className={representClasses}>
+
+      <div className='relative card card-stacked-shadow card-course flex expanded-horizontal'>
+        <div className='flex flex-column items-center overflow-hidden br2 br--left pa4 bg-white navy pointer relative z-1 card-course-inner'>
+          <div className='fa fa-play w3 h3 f3 absolute z-1 gray bg-white-70 o-0 items-center justify-center br-pill pointer card-play-btn'></div>
+          <div className='mw5 mt3 center ph3'>
+            <img alt='' src={imgCourseCard}/>
+          </div>
+          <h3 className='f3 tc mt4 mb2 avenir fw5'>Introduction to RxJS Marble Testing Two lines headline</h3>
+          <div className='f6 dark-gray o-50 mb4 tc'>Joe Maddalone</div>
+          <div className='flex justify-between self-stretch items-center'>
+            <div className='f6 dark-gray o-50'>12 lessons</div>
+            <div className='f6 orange lh-title ttu fw6 ph3 pv1 br-pill bg-tag-orange tracked'>course</div>
+          </div>
+        </div>
+        <div className='relative w-100 z-1 overflow-hidden pv3 br2 br--right bg-tag-gray' style={{
```

Comment:
No, just for now I'm not sure how we will keep the width of Card - with parent grid elements or just in component styles. So for now I leave it as inline style.